db/schema.rb

```diff
@@ -10,9 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_02_10_231042) do
+ActiveRecord::Schema.define(version: 2021_03_03_102638) do
 
-  create_table "bmap_flowcell", primary_key: "id_bmap_flowcell_tmp", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
+  create_table "bmap_flowcell", primary_key: "id_bmap_flowcell_tmp", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
```

Comment:
do you know what it is that's causing this `id: :integer` to suddenly appear?

Reply:
No idea. Things like this happen when you rerun migrations sometimes. It's difficult to keep track of. I have run it locally and against UAT and all is working fine.